source/tyr/tests/integration/users_test.py

```diff
@@ -291,44 +207,49 @@ def test_add_user_with_invalid_geojson(mock_rabbit, invalid_geojsonfixture):
     assert status == 400
     assert mock_rabbit.call_count == 0
 
+
 def test_add_user_with_plus(mock_rabbit):
     """
     creation of a user with a "+" in the email
     """
     user = {'login': 'user1+test@example.com', 'email': 'user1+test@example.com'}
     resp = api_post('/v0/users/', data=json.dumps(user), content_type='application/json')
+
     def check(u):
-        for k,_ in user.iteritems():
+        for k, _ in user.iteritems():
```

Comment:
Minor: if you want check the perfect equality of two dicts, just use `cmp() == 0` 
https://docs.python.org/2/library/functions.html#cmp